client/hwctl.1

```diff
@@ -0,0 +1,48 @@
+.Dd December 5, 2024
+.Dt HWCTL 1
+.Os Ubuntu
+.Sh NAME
+.Nm hwctl
+.Nd CLI tool to check hardware certification status
+.Sh SYNOPSIS
+.Nm
+.Sh DESCRIPTION
+The
+.Nm
+utility checks the
+.Em Ubuntu Hardware Certification
+status of your system.
+.Pp
+.Nm
+verifies whether your exact machine model
+or a similar system has been certified and displays which Ubuntu
+releases the certification covers for it.
+For more information about the
+.Em Ubuntu Hardware Certification Program ,
+visit
+.Lk https://ubuntu.com/certified "the certification website" .
+.Pp
+Note that root privileges are typically required to run this command.
+.Sh OPTIONS
+The
+.Nm
+utility does not accept any command-line options.
+.Sh ENVIRONMENT
+.Bl -tag -width "HW_API_URL" -compact
+.It Ev HW_API_URL
+Specifies the hardware-api server URL that
+.Nm
+uses for certification queries. The default value is
+.Pa https://hw.ubuntu.com .
```

Comment:
```suggestion
. Lk https://hw.ubuntu.com .
```

Reply:
But it's already shown as a link on the man page
![image](https://github.com/user-attachments/assets/8ecf4744-eb18-4d41-991a-6e0319cd986c)

Reply:
Yes but .Pa says "this is a path". The link is rendered by your terminal emulator by matching text.